### DISCUSSION TITLE
Get IAU constellation boundaries coordinates?

### DISCUSSION THREAD

**Discussion by tergvey** [skyfielders/python-skyfield#814] [Q&A] [answered]
Thank you for putting so much efforts this package. I am sorry I'm not in a position where I can contribute myself to the project.

I've read the documentation quite a few times, am I correct that one can not get the IAU boundaries from skyfield.api.load_constellation_map(), only the constellation lines? It feels like I'm missing something since you can get which constellation an object is in with constellation_at()? 

I can add that my use for this was to get the constellation boundaries along the ecliptic for a specific time period, so to dynamically could see how it varies through the ages. 

Thank you!

**Comment by brandon-rhodes** [comment, marked as answer]
Skyfield does not include any constellation lines, since there is no international standard for how to draw them. Take a look at Stellarium's cultural sky database for different ways to draw the constellation lines.

The boundaries, or "boxes", that define the constellations are not available though a Skyfield function, but in a built-in data file that's shipped with Skyfield, that's based on the <http://cdsarc.u-strasbg.fr/ftp/VI/42/data.dat> data file. Here's how the built-in grid can be loaded:

```
from skyfield.functions import load_bundled_npy
a = load_bundled_npy('constellations.npz')
print(a['sorted_ra'])
print(a['sorted_dec'])
print(a['radec_to_index'])
print(a['indexed_abbreviations'])
```

If you are interested in constellation boundaries, my guess is that you will want to start with the official boundary file, as it is more official than Skyfield itself.
